docs/develop/01_sdk/02_cookbook/01_dids/06_full_did_tx.md

```diff
@@ -0,0 +1,51 @@
+---
+id: full-did-batch
+title: Build DID Extrinsics
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import FullDidSignTx from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/did/08_full_did_sign_tx.ts';
+import FullDidBatch from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/did/08_full_did_batch.ts';
+
+A DID can be used to sign extrinsics.
+But not every extrinsics can be sign using a DID.
+The Spiritnet blockchain offers two flavours of extrinsics.
+
+The first flavour can only be called using an account.
+These extrinsic require fee payment and need therefore be send from an account with sufficient balance.
+
+The second flavour is callable from a DID.
+Since every extrinsic requires fees to be paid, this flavour needs to be wrapped inside a normal account extrinsic.
+The account will pay all fees and deposit that will be required.
+
+This document describes how to sign the DID extrinsics.
+The KILT SDK provides two functions for signing DID extrinsics.
+The first function signs a single extrinsic while the second one batches multiple extrinsics together.
+
+## Single extrinsics
+
+To build a single extrinsic, you need to provide:
+
+* the DID that should be the origin of the extrinsic
```

Comment:
```suggestion
* the DID that wants to sign the extrinsic (also called *origin* of the extrinsic)
```

Reply:
Maybe the order of these elements can match the order of the parameters in the function signature? That would make it a bit easier for the users to follow the two at the same time.